### ISSUE TITLE
[Visual QA] Product Detail page

### ISSUE THREAD

**Issue by mindywagner** [vigetlabs/npm#100]
**All:**
- Breadcrumb links above page title (Products > Audio) are missing
- Available for: XXX is missing
- Spec tab text - all links should be #0A6EB9 (currently inactive is grayed out)
- Buttons are the "Button with icon" variety seen in the Styles > Buttons board, and both need the doc icon
![Screen Shot 2019-11-01 at 2 50 30 PM](https://user-images.githubusercontent.com/67819/68048451-f8cde000-fcb6-11e9-9455-7b31e3c3ad6c.png)

- Writing a separate ticket for Featured Products element
- Do they need to be able to add 4 featured products? I thought we capped it at 2. Looks like a 2nd row is showing up with nothing in it. 

Buildout:
<img width="1169" alt="Screen Shot 2019-11-01 at 3 07 44 PM" src="https://user-images.githubusercontent.com/67819/68049486-6549de80-fcb9-11e9-90ae-c792bab8de28.png">

- Related Work element is missing
- Samples appear to be missing

**Desktop:**
- Padding between product intro text & button should be 40px
- Padding between the navigation/header element and the missing Breadcrumb link should be 70px (currently looks like it's 100px)
- Similarly, padding between the end of the product intro text and the top of the Specs headline should be 70px; currently ~235px
- Space between the Specs headline and the top of the Tab text should be 30px; currently closer to 40px
- Spec text that isn't a label (ex: 15 seconds, 30 seconds) is 16px 
- When there's no image added, the text should widen to 840px
- Some of the sample thumbnail image is getting cut off on the left/right (ex: https://live-national-public-media-v2.pantheonsite.io/products/mid-roll/)

_________________

**Midsize:**
- Can we put the "Contact Us" button below the image when this element collapses for smaller screens?

_________________

**Mobile:**
- Padding between the tab nav line and the "Duration" label should be 30px - currently is closer to 40px

**Comment by gracecanfield** [comment]
@mindywagner Heads up that I reviewed and removed a couple items from your list that are either already captured elsewhere, or are just content entry things.

**Comment by justinvandenberg** [comment]
@mindywagner sample thumbnail image is just an issue with the image uploaded.

**Comment by mindywagner** [comment]
I tried uploading with an image sized correctly, and it's still doing it. Image attached. 

![Product Sample Thumbnail](https://user-images.githubusercontent.com/67819/68512853-7cec0e80-0247-11ea-91d9-6201ead55b7f.png)

**Comment by justinvandenberg** [comment]
@mindywagner I figured there would be padding in the images, but I can add that to the container instead.

**Comment by mindywagner** [comment]
There's padding built into that image 

![Screen Shot 2019-11-08 at 4 55 27 PM](https://user-images.githubusercontent.com/67819/68513268-98a3e480-0248-11ea-96ee-0e2f65a78f8e.png)

**Comment by mindywagner** [comment]
### Still not done from above - desktop:

- [ ] Padding between the navigation/header element and the missing Breadcrumb link should be 70px (currently looks like it's 100px) Buildout: 
![Screen Shot 2019-11-18 at 1 44 04 PM](https://user-images.githubusercontent.com/67819/69080250-993b3880-0a09-11ea-90cb-79910461428d.png)

- [ ] Padding between the end of the product intro text and the top of the Specs headline should be 70px total; currently ~150px (100 from the spec tab element, 50 from the top section element)
![Screen Shot 2019-11-18 at 1 47 41 PM](https://user-images.githubusercontent.com/67819/69080450-05b63780-0a0a-11ea-87ad-eb8f2a9c954b.png)

- [ ] Spec text in the spec table that isn't a light gray label (ex: 15 seconds, 30 seconds) should be 16px. Label should stay 14px.
![Screen Shot 2019-11-18 at 1 49 27 PM](https://user-images.githubusercontent.com/67819/69080581-43b35b80-0a0a-11ea-98e6-65cb21f63dca.png)
 
- [ ] When there's no image added, the text should widen to 840px

**One new one:**
- [ ] Download Specs & Creative Guidelines buttons need their hover state

### Mobile:

- [ ]  Reduce padding above Products eyebrow element to 20px (currently 50) - this should apply to all mobile hero areas/page titles. I think I have a ticket in for this already, but noting it here just in case.

- [ ] Spacing between intro text and spec tabs needs reducing, similar to desktop. Total padding between both elements should be 50px. Currently doubling to 100px. I achieved the right padding by removing top padding on specs and changing bottom padding on intro text to 20px instead of 30px. Not sure this is the way to go, but wanted to share.

**Comment by justinvandenberg** [comment]
@mindywagner I can't recall where it was discussed (I think it's in the figma notes?) but we decided to nix the second part of breadcrumbs because products can be related to multiple platforms.

**Comment by gracecanfield** [comment]
That's correct - that was me that specified it in the comp annotations. Mindy, let us know if you'd prefer to handle a different way, but we felt that it would be super weird to have, like a comma-separated list of platforms in a breadcrumb.

**Comment by mindywagner** [comment]
fine with me!
